my/heartbox.scad

```diff
@@ -0,0 +1,37 @@
+/// Create a Heart box with 2 circle and a square + minkowski function
+
+$fa = 0.01;
+$fs = 0.5;
+heart_box(20,15,0.5);
+
+module heart(s,h) {
+    linear_extrude(height=h) {
+      union() {
```

Comment:
Please try to match the code style in the rest of the repository! It indents with tabs rather than spaces, always uses brackets (on the same line) even if the subclause is only 1 line.